DataRepo/templates/upload.html

```diff
@@ -1,50 +1,109 @@
 {% extends "base.html" %}
 {% load static %}
 {% block content %}
-<div>
-    <h4>Animal, Sample, &amp; Accucor Data Submission Process</h4>
-    <hr />
+    <div>
+
+        <h4 id="animal-sample-accucor-data-submission-process">Animal, Sample, &amp; Accucor Data Submission Process</h4>
+        <h5 id="create-a-sample-sheet">Create a Sample Sheet</h5>
+        <p>An important part of TraceBase is collecting and maintaining a consistent set of metadata for each sample.</p>
+        <ol>
+            <li>Make a copy of the <a href="https://docs.google.com/spreadsheets/d/1To3495KxJkAtnAD9KVdzc162zKbNiYuPEVyPseQPjMQ/copy">Tracebase Animal and Sample Table Template</a>.
+            </li>
+            <li>Fill in the <em>Animals</em> sheet <a class="badge bg-secondary" data-bs-toggle="collapse" data-bs-target="#fill-animals">more info</a>
```

Comment:
NB. Oh yeah.  I would also name the expansion links more succinctly or even use those triangle things, if you felt feisty.
```suggestion
            <li>Fill in the <em>Animals</em> sheet <a class="badge bg-secondary" data-bs-toggle="collapse" data-bs-target="#fill-animals">more</a>
```

Reply:
I originally looked at using the triangles, but making them animated was more than I wanted to do right now. I can change these to just "more", however.